host/sgxload.c

```diff
@@ -887,10 +937,9 @@ oe_result_t oe_sgx_delete_enclave(oe_enclave_t* enclave)
         OE_RAISE(OE_INVALID_PARAMETER);
 
     /* free allocate memory. */
-    result = _sgx_free_enclave_memory(
-        (void*)enclave->addr, enclave->size, enclave->simulate);
-
+    OE_RAISE(
```

Comment:
Coding style: We only do OE_RAISE(error-code).
Use OE_CHECK instead.

Reply:
addressed